From e8c02a2fb79ec1d49e63ea082ffa5353cac754e8 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Mon, 8 Feb 2016 19:18:32 +0100
Subject: [PATCH] wizard: fix the broken "Show log" widget

It looks like that GtkScrolledWindow widget is no longer expanded
automatically.

Fixes: rhbz#1303326

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/gui-wizard-gtk/wizard.glade | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gui-wizard-gtk/wizard.glade b/src/gui-wizard-gtk/wizard.glade
index 1c45bd9..b691e4d 100644
--- a/src/gui-wizard-gtk/wizard.glade
+++ b/src/gui-wizard-gtk/wizard.glade
@@ -812,6 +812,8 @@
               <object class="GtkScrolledWindow" id="scrolledwindow6">
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
+                <property name="hexpand">True</property>
+                <property name="vexpand">True</property>
                 <property name="shadow_type">out</property>
                 <child>
                   <object class="GtkTextView" id="tv_event_log">
-- 
1.8.3.1

